server/protocols/webadmin/webadmin-mailqueue/src/main/java/org/apache/james/webadmin/service/ClearMailQueueTask.java

```diff
@@ -110,7 +111,9 @@ public Result run() {
             this.initialCount = Optional.of(getRemainingSize(queue));
             this.queue = Optional.of(queue);
             queue.clear();
-            this.lastAdditionalInformation = details();
+            this.lastAdditionalInformation = Mono.from(detailsReactive())
+                .subscribeOn(ReactorUtils.BLOCKING_CALL_WRAPPER)
+                .block();
```

Comment:
Why?

Reply:
Ah, I should not subscribeOn blocking wrapper. My miss